### ISSUE TITLE
Telegram is not optimized for working in split-screen mode

### ISSUE THREAD

**Issue by seawithfire** [telegramdesktop/tdesktop#28797]
### Is your feature request related to a problem?

I usually run Telegram alongside Photoshop when I'm working. In this case, to take up the least amount of space, I have to reduce the width of Telegram as much as possible, which hides the "profiles" and when a new message arrives, I don't know who sent the message (I closed the notification) and have to check it manually.

Currently, the only way to have both the chat and the profile column is with one of the first two photos. Is it possible to change it to have a third mode?

![ass](https://github.com/user-attachments/assets/93d78e00-96e1-4f39-85bc-8c7264e93ae6)

### Describe the solution you'd like

i think its easy for developers to the third mode. i hope they see this thread

### Describe alternatives you've considered

no any alternative way

**Comment by ilya-fedin** [comment]
> i think its easy for developers to the third mode.

No, this thinking is wrong, sorry. For that to happen, the widgets have to be reworked to become more adaptive. As-is, just allowing lesser sizes for the messages area would lead to content getting out of the bubbles and maybe even the window.

**Comment by Aokromes** [comment]
> ### Is your feature request related to a problem?
> I usually run Telegram alongside Photoshop when I'm working. In this case, to take up the least amount of space, I have to reduce the width of Telegram as much as possible, which hides the "profiles" and when a new message arrives, I don't know who sent the message (I closed the notification) and have to check it manually.
> 
> Currently, the only way to have both the chat and the profile column is with one of the first two photos. Is it possible to change it to have a third mode?
> 
> ![ass](https://private-user-images.githubusercontent.com/150816470/398445972-93d78e00-96e1-4f39-85bc-8c7264e93ae6.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUwNjU0MjksIm5iZiI6MTczNTA2NTEyOSwicGF0aCI6Ii8xNTA4MTY0NzAvMzk4NDQ1OTcyLTkzZDc4ZTAwLTk2ZTEtNGYzOS04NWJjLThjNzI2NGU5M2FlNi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyNFQxODMyMDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mZGU0ZWIxMzg0ODdlY2NmZDRkMzhlNGEyNzgzMDI2MmM2OTk4YWIxNzJiODQ0NmU2NzFiYmJmYjNlMjFhMTZjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.MAlD1l3mFMpF4-KhhGuLOq209wt75-G1ecoO7nIYUB0)
> 
> ### Describe the solution you'd like
> i think its easy for developers to the third mode. i hope they see this thread
> 
> ### Describe alternatives you've considered
> no any alternative way
> 
> ### Additional context
> _No response_

use scaling under 100%

**Comment by seawithfire** [comment]
> use scaling under 100%

didnt work. things just get supersmall. still if i make width smaller, the profile area disappear

**Comment by seawithfire** [comment]
> No, this thinking is wrong, sorry. For that to happen, the widgets have to be reworked to become more adaptive. As-is, just allowing lesser sizes for the messages area would lead to content getting out of the bubbles and maybe even the window.

so maybe they can add an extra column for chats avatars (just avatar, no name no text, like the ones for foldres)

**Comment by ilya-fedin** [comment]
> so maybe they can add an extra column for chats avatars

Not sure what you mean. That sounds just like what's on your screenshot and I was commenting exactly that. You can't just add that column as that would lesser message area width and in turn lead to content getting out of the bubbles. Thus a bigger redesign is needed to achieve that.

**Comment by seawithfire** [comment]
> Not sure what you mean. That sounds just like what's on your screenshot 

the third mode in screenshot is not real! its what i want to be. now the only split that have both chats list and msg list are 1 or 2 ... the 3 is not possible. i want the third

**Comment by ilya-fedin** [comment]
> the third mode in screenshot is not real! its what i want to be.

I understand. And I explained multiple times that it's not simple at all, it needs widgets reworking so the content doesn't get out of bubbles.
